cmd/cstorvolumeclaim/start.go

```diff
@@ -27,23 +31,27 @@ import (
 	clientset "github.com/openebs/maya/pkg/client/generated/clientset/versioned"
 	informers "github.com/openebs/maya/pkg/client/generated/informers/externalversions"
 	ndmclientset "github.com/openebs/maya/pkg/client/generated/openebs.io/ndm/v1alpha1/clientset/internalclientset"
-	"github.com/openebs/maya/pkg/signals"
+	leader "github.com/openebs/maya/pkg/kubernetes/leaderelection"
 	kubeinformers "k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 )
 
 var (
-	masterURL  string
-	kubeconfig string
+	masterURL              string
+	kubeconfig             string
+	leaderElectionLockName = "cvc-controller-leader"
+)
+
+// Command line flags
+var (
+	leaderElection          = flag.Bool("leader-election", true, "Enables leader election.")
+	leaderElectionNamespace = flag.String("leader-election-namespace", "", "The namespace where the leader election resource exists. Defaults to the pod namespace if not set.")
```

Comment:
...leader election resource ... are we talking about the `Lease`  resource here. Putting it in a `i.e.` kind of statemnt will make more sense for the first time viewers like me.